src/components/auctions/HighestVolumeAuctions/index.tsx

```diff
@@ -0,0 +1,292 @@
+    }
+  }
+  &:last-child {
+    display: none;
+    @media (min-width: ${(props) => props.theme.themeBreakPoints.xl}) {
+      display: flex;
+    }
+  }
+`
+
+const StyledThCell = styled.div`
+  display: flex;
+  flex-grow: 1;
+  align-items: center;
+  justify-content: space-between;
+  font-weight: bold;
+  font-size: 18px;
+  padding: 7px 10px;
+  &:nth-child(2) {
+    justify-content: center;
+  }
+  &:first-child {
+    padding-left: 30px;
+    justify-content: flex-end;
+  }
+`
+
+const SmallLbl = styled.div`
+  font-size: 11px;
+  color: ${(props) => props.theme.dropdown.item.color};
+  opacity: 0.5;
+  font-weight: normal;
+`
+
+interface HVAuctionsProps {
+  highestVolumeAuctions?: Maybe<AuctionInfo[]>
+}
+
+const HighestVolumeAuctions = ({ highestVolumeAuctions }: HVAuctionsProps) => {
+  const noAuctions = !highestVolumeAuctions || highestVolumeAuctions?.length === 0
+  return (
+    <Wrapper>
+      <SectionTitle style={{ display: 'block' }}>Highest Volume Auctions</SectionTitle>
+      {noAuctions ? (
+        <EmptyContentWrapper>
+          <InfoIcon />
+          <EmptyContentText>No auctions.</EmptyContentText>
+        </EmptyContentWrapper>
+      ) : (
+        <StyledTable>
+          <StyledTHead>
+            <StyledThCell>Sell Ammount</StyledThCell>
+            <StyledThCell>Buy Ammount</StyledThCell>
+            <StyledThCell>USD Volume</StyledThCell>
+            <StyledThCell>End Date</StyledThCell>
+          </StyledTHead>
+          {highestVolumeAuctions.map((auction, index) => (
+            <StyledTr key={index}>
+              <StyledTd>
+                <DoubleLogo
+                  auctioningToken={{
+                    address: auction.addressAuctioningToken,
+                    symbol: auction.symbolAuctioningToken,
+                  }}
+                  biddingToken={{
+                    address: auction.addressBiddingToken,
+                    symbol: auction.symbolBiddingToken,
+                  }}
+                  size="35px"
+                />
+              </StyledTd>
+              <StyledTd>
+                {auction.symbolAuctioningToken} / {auction.symbolBiddingToken}
+              </StyledTd>
+              <StyledTd>
+                <BottomIconNetwork />
+                <SmallLbl>{getChainName(parseInt(auction.chainId.toString()))}</SmallLbl>
+              </StyledTd>
+              <StyledTd>
+                <span>Sell Ammount:&nbsp;</span>
+                {auction.order.volume + ' ' + auction.symbolAuctioningToken}
+              </StyledTd>
+              <StyledTd>
+                <span>Buy Ammount:&nbsp;</span>
+                {auction.order.volume + ' ' + auction.symbolAuctioningToken}
+              </StyledTd>
+              <StyledTd>
+                <span>USD Volume:&nbsp;</span>
+                {'$' + auction.order.volume}
+              </StyledTd>
+              <StyledTd>{moment(new Date()).format('DD/MM/YYYY')}</StyledTd>
```

Comment:
let's just use Date's `toLocaleDateString` for formatting and also remove momentjs library from the dependencies to reduce the bundle size